README.md

```diff
@@ -153,28 +153,28 @@ pnpm format
 
 Die Regeln für Prettier können in der `.prettierrc` Datei konfiguriert werden. Weitere Informationen finden sich in der [Prettier Dokumentation](https://prettier.io/docs/en/index.html).
 
-## Auf Vercel bereitstellen
+## Infrastruktur
+
+
+### Auf Vercel bereitstellen
 
 Der einfachste Weg, die Next.js App bereitzustellen, ist die Verwendung der [Vercel Plattform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) von den Erstellern von Next.js.
 
 Die [Next.js Bereitstellungsdokumentation](https://nextjs.org/docs/app/building-your-application/deploying) sollte für weitere Details angesehen werden.
 
+### Zugang zu Hitabis VM
 
-## Architektur
-
+Anmelden mit 
 
-### Komponenten
+```
+ssh -i </pfad/zu/ssh/key> htw@88.99.127.140
```

Comment:
Bitte keine Zugänge in Git. Bitte Änderungen bitte Purgen.

Reply:
entfernt